doc/examples/viz_ui.py

```diff
@@ -143,14 +143,17 @@ def translate_green_cube(i_ren, obj, slider):
     value = slider.value
     cube_actor_2.SetPosition(value, 0, 0)
 
-
 line_slider = ui.LineSlider2D(initial_value=-2,
                               min_value=-5, max_value=5)
 
 line_slider.add_callback(line_slider.slider_disk,
                          "MouseMoveEvent",
                          translate_green_cube)
 
+line_slider.add_callback(line_slider.slider_line,
```

Comment:
Can you do the same for the disk slider also? Let's merge both these into this PR.